$NetBSD: patch-CMakeLists.txt,v 1.4 2017/12/01 19:22:12 adam Exp $

Disable tests: they require missing files from standard LLVM build (lit tool)

--- CMakeLists.txt.orig	2017-07-11 11:37:35.000000000 +0000
+++ CMakeLists.txt
@@ -176,10 +176,6 @@ add_definitions( -D_GNU_SOURCE )
 
 add_subdirectory(docs)
 add_subdirectory(lib)
-add_subdirectory(test)
-if (POLLY_GTEST_AVAIL)
-  add_subdirectory(unittests)
-endif ()
 add_subdirectory(tools)
 add_subdirectory(cmake)
 # TODO: docs.
